engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/views/product-repositories.html

```diff
@@ -179,6 +179,12 @@
                   {{ repository.content_counts.deb || 0 }} deb Packages
                 </a>
               </div>
+
+              <div>
```

Comment:
I think the reason that we didn't show this before is because it isn't applicable to all repos. Given my above suggestion around the RepositoryTypeManager I think we could add a new parameter to this repositories api called `supports_errata` or similar, and use that to determine if we should show this line